packages/forms-web-app/__tests__/unit/controllers/common/your-email-address.test.js

```diff
@@ -36,30 +39,35 @@ describe('controllers/full-appeal/submit-appeal/enter-code', () => {
 			const testEmail = 'iamnoone@@planninginspectorate.gov.uk';
 			req.params.id = testId;
 			req.body['email-address'] = testEmail;
+			getUserByEmail.mockResolvedValue({
+				_id: '64abf5b59beabb0019775c23',
+				email: 'admin1@planninginspectorate.gov.uk',
```

Comment:
Think Hawley mentioned on another PR if  we could use @example.com for emails in tests